prometheus-to-sd/main.go

```diff
@@ -45,8 +46,16 @@ var (
 		"The resolution at which prometheus-to-sd will scrape metric descriptors from Stackdriver.")
 	apioverride = flag.String("api-override", "",
 		"The stackdriver API endpoint to override the default one used (which is prod).")
-	source = flags.Uris{}
-	podId  = flag.String("pod-id", "machine",
+	source    = flags.Uris{}
+	authToken = flag.String("auth-token", "",
```

Comment:
What if instead of making it as a global flags, we move this to per source configuration? In such case if different components have different ways to authorize we can use different tokens.

Reply:
Fixed.